example/bookstore/v1/bookstore.yaml

```diff
@@ -44,3 +37,12 @@ resources:
 #       - "Publisher"
 #     methods:
 #       read: {}
+objects:
+  - kind: "Details"
+    properties:
+      edition:
+        type: STRING
+        number: 1
+      supportedEReaders:
```

Comment:
I don't think we've talked about this much, but I think fields are probably better off described with underscores vs camel case.

My reasoning: it's hard to tell what is capitalized as part of an abbreviation, vs what is capitalized as a separator from the previous word. Thoughts?

Reply:
we can translate the property name as part of the transformation to openapi, and proto -> json should handle it flawlessly as well.

Reply:
Agree with that. As long as we don't try to reimplement underscore -> camelcase conversions, I'm good with this.